app/javascript/controllers/organization_report_controller.js

```diff
@@ -0,0 +1,67 @@
+import ApplicationController from './application_controller'
+import events from '../events'
+import { toArray } from '../src/utils'
+import moment from 'moment'
+
+export default class extends ApplicationController {
+  static targets = ['campaign', 'useSelectedCampaignDates', 'dateRangePicker']
+
+  connect () {
+    super.connect()
+    this.currentStartDate
+    this.currentEndDate
+  }
+
+  updateDateRange () {
+    this.currentStartDate = moment(
+      this.dateRangePickerTarget.dataset.startDate,
+      'MM/DD/YYYY'
+    ).utc()
+    this.currentEndDate = moment(
+      this.dateRangePickerTarget.dataset.endDate,
+      'MM/DD/YYYY'
+    ).utc()
+
+    let newStartDate = this.currentStartDate
+    let newEndDate = this.currentEndDate
+    let minDate = this.getMinMaxDates().minDate
+    let maxDate = this.getMinMaxDates().maxDate
+
+    if (this.useSelectedCampaignDatesTarget.checked) {
+      newStartDate = minDate
+      newEndDate = maxDate
+    }
+
+    jQuery(this.dateRangePickerTarget)
```

Comment:
Do we have to use jQuery? I’m assuming yes but wanted to call that out.

Reply:
Yes we do. This is required to trigger the setStartDate